lib/iex/lib/iex/info.ex

```diff
@@ -0,0 +1,82 @@
+defprotocol IEx.Info do
+  @spec info(term) :: [{atom, iodata}]
```

Comment:
Also these atom keys are being used as strings, why not use a map with string keys?